src/components/TodoComponents/Todo.js

```diff
@@ -1,14 +1,34 @@
 import React from 'react'
 
-const Todo = props =>{
+export default function Todo(props){
+
+    let classNames = 'todo';
+    if (props.todo.completed){
+        classNames = 'completed'; //for whatever reason todo.completed doesnt work
+    }
```

Comment:
Yes, `todo.completed` will not work because the `todo` variables actually comes from the `props` variable that you passed in as a parameter of this component `Todo`.